src/App.css

```diff
@@ -64,8 +70,16 @@ p {
   width: 60%;
 }
 
+.category,
+.author {
+  color: #1e51dcab;
+}
+
 .card-col2 {
   width: 20%;
+  display: flex;
+  align-items: center;
+  gap: 1em;
```

Comment:
- [ ] It would be better to try and remain consistent with the background color of your `header` and `buttons` like in the original design. this will  improve the aesthetic aspect of your design

![image](https://github.com/Mylo16/book-store/assets/98453979/d0f5c83d-e5f7-42e3-8054-7ce472346cf1)

![image](https://github.com/Mylo16/book-store/assets/98453979/8d39d0df-e08c-4fe1-9427-1d12b267d4d5)

### Checkout the original design

![image](https://github.com/Mylo16/book-store/assets/98453979/78797f6e-e78a-411f-bbac-eba0534a6e3b)